nomad/structs/structs.go

```diff
@@ -10834,6 +10842,9 @@ type DeploymentStatusUpdate struct {
 
 	// StatusDescription is the new status description of the deployment.
 	StatusDescription string
+
+	// UpdatedAt is the time of the update
```

Comment:
I love being specific with all of these bare int64 timestamp fields... especially since they're not entirely consistent. :sob: 
```suggestion
	// UpdatedAt is the time of the update, stored as UnixNano
```